controls/radgridview/events/column-resize-event.md

```diff
@@ -1,128 +1,173 @@
 
-{{region gridview-column-resize-event_1}}
+The **GridViewAutoGeneratingColumnEventArgs** class provides you with a few useful properties:
 
-	gridView.ColumnWidthChanging += new EventHandler<ColumnWidthChangingEventArgs>(gridView_ColumnWidthChanging);
-	gridView.ColumnWidthChanged += new EventHandler<ColumnWidthChangedEventArgs>(gridView_ColumnWidthChanged);
-{{endregion}}
+* **Cancel**: By setting this to **True**, you can cancel the creation of the current column.
 
-#### __VB.NET__
+* **Column**: The column that is being generated.
 
-{{region gridview-column-resize-event_2}}
+* **ItemPropertyInfo**: Information about the property, based on which the column is generated.
 
-	AddHandler gridView.ColumnWidthChanging, AddressOf gridView_ColumnWidthChanging
-	AddHandler gridView.ColumnWidthChanged, AddressOf gridView_ColumnWidthChanged
-{{endregion}}
+**Example 2** demonstrates how you can cancel the creation of a specific column:
 
-## ColumnWidthChanging event
+#### __[C#] Example 2: Cancel the creation of DateTime columns__
 
-The __ColumnWidthChanging__ event occurs when the column is being resized. If it is canceled the column does not change its width and the __ColumnWidthChanged__ event does not fire. The event handler receives the following two arguments:
+	{{region gridview-column-resize-event_3}}
+    private void gridView_AutoGeneratingColumn(object sender, GridViewAutoGeneratingColumnEventArgs e)
+    {
+        if (e.ItemPropertyInfo.PropertyType == typeof(DateTime))
+        {
+            e.Cancel = true;
+        }
+    }
+	{{endregion}}
 
-* The __sender__ argument contains the __RadGridView__. This argument is of type object, but can be cast to the __RadGridView__ type.
+#### __[VB.NET] Example 2: Cancel the creation of DateTime columns__
 
-* A __ColumnWidthChangingEventArgs__ object. This object has the following properties: 
+	{{region gridview-column-resize-event_4}}
+	Private Sub gridView_AutoGeneratingColumn(sender As Object, e As GridViewAutoGeneratingColumnEventArgs)
+		If e.ItemPropertyInfo.PropertyType = GetType(DateTime) Then
+			e.Cancel = True
+		End If
+	End Sub
+	{{endregion}}
 
-* __Cancel__ - gets or sets the value indicating whether the event should be canceled.
+## ColumnReordering
 
-* __Column__ - the column that is being resized
+The __ColumnReordering__ event occurs when RadGridView's columns are being reordered either programmatically or through the UI.
 
-* __HorizontalChangeWidth__ - a double value which indicates the number of pixels that would be added/subtracted to the column width. If you shrink the width the value will be negative, e.g. -10
+>The user can reorder columns only if the __CanUserReorderColumns__ is set to __True__ (which is the default value). 
 
-* __OriginalWidth__ - a double value which presents the original width of the column
+Through the __ColumnReorderingEventArgs__ you can access the following properties:
```

Comment:
Seems that you have made a slight mistake as this is the Reordering event, not the resizing one. Please change to your likings.